back-end/routes/pendingTransactions.js

```diff
@@ -3,20 +3,24 @@ const router = express.Router();
 const { PrismaClient } = require("@prisma/client");
```

Comment:
Should this route be moved to back-end/routes/user.js

Reply:
what do you mean by this?